packages/hash/frontend/src/pages/login.page/login-modal.tsx

```diff
@@ -208,6 +208,8 @@ export const LoginModal: FunctionComponent<LoginModalProps> = ({
 
   const defaultLoginIdentifier = useMemo(() => {
     const { email, shortname } = router.query;
+    // @todo how to handle empty email
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
Optional (in all cases):

```suggestion
    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing -- @todo how to handle empty email
```

ESLint auto-removes `eslint-disable-next-line` when it is no longer needed. If this happens, the current `@todo` comment will be preserved and may confuse devs. Combining two lines into one  keeps comment hard-linked to the ESLint directive.